src/electionguard_cli/cli_steps/key_ceremony_step.py

```diff
@@ -22,11 +22,6 @@ def run_key_ceremony(self, guardians: List[Guardian]) -> ElectionJointKey:
             "mediator_1", guardians[0].ceremony_details
         )
         KeyCeremonyOrchestrator.perform_full_ceremony(guardians, mediator)
```

Comment:
`perform_full_ceremony` does all the same stuff as the 3 commands below, so removed duplicate logic.  May need to switch back to the individual steps when we get the CLI to perform a key ceremony in steps, but can cross that bridge when we get to it.